### DISCUSSION TITLE
Redirect to external URL?

### DISCUSSION THREAD

**Discussion by lephyrius** [leptos-rs/leptos#1491] [Q&A] [answered]
I'm doing an OAuth Flow but seems like the "Redirect" component only accepts internal path links. 
Any way of making it accept a full URL?

**Comment by gbj** [comment, marked as answer]
Honestly in this case I'd just use [`window.location`](https://docs.rs/web-sys/latest/web_sys/struct.Location.html#method.assign) —
```rust
_ = window().location().assign("https://www.my-cool-site.org")
```
